react/Header/Navigation/Navigation.less

```diff
@@ -54,3 +56,16 @@
     box-shadow: inset 0 -2px @sk-white;
   }
 }
+
+.link_smallText {
+  font-size: 16px;
```

Comment:
If we retain this behaviour I would recommend using the `Text` component and using the boolean flags to select another hierarchy element conditionally rather than apply bespoke font sizing. It keeps all `font-size` declarations inside of `Text` which is desirable